File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.cs
using System;

public class Solution {
    public int solution(string myString, string pat) {
        char[] arr = myString.ToCharArray();
        for(int i = 0; i < arr.Length; i++)
        {
            if(arr[i] == 'A') arr[i] = 'B';
            else arr[i] = 'A';
        }
        string answer = new string(arr);
        return answer.Contains(pat) ? 1 : 0;
    }
}

File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>
using namespace std;

int solution(string myString, string pat) {
    for(char &c : myString)
    {
        if(c == 'A') c = 'B';
        else c = 'A';
    }
    if(myString.find(pat) != string::npos) return 1;
    return 0;
}